composer.json

```diff
@@ -6,14 +6,14 @@
     "require": {
         "php": ">=7.2",
         "spryker/console": "^4.0.0",
-        "spryker/development": "^3.0.0",
         "spryker/monolog": "^2.0.0",
         "spryker/symfony": "^3.0.0",
         "spryker/transfer": "^3.4.0",
         "spryker/util-encoding": "^2.0.0",
         "spryker-middleware/logger": "^1.0.0"
     },
     "require-dev": {
+        "spryker/development": "^3.0.0",
         "spryker/calculation": "*",
         "spryker/cart": "*",
         "spryker/code-sniffer": "*",
```

Comment:
please remove "spryker/transfer": "*" from "require-dev", it's already requirement

Reply:
done